packages/deployments/contracts/contracts/core/connext/facets/ProposedOwnableFacet.sol

```diff
@@ -67,6 +67,23 @@ contract ProposedOwnableFacet is BaseConnextFacet, IProposedOwnable {
 
   event Unpaused();
 
+  // ============ Modifier ============
+  /**
+   * @notice Throws if the ownership delay has not elapsed
+   */
+  modifier ownershipDelayElapsed() {
+    // Ensure delay has elapsed
+    if ((block.timestamp - s._proposedOwnershipTimestamp) <= delay())
+      revert ProposedOwnableFacet__ownershipDelayElapsed_delayNotElapsed();
+    _;
+  }
+
+  modifier delayElapsed(uint256 start) {
+    // Ensure delay has elapsed
+    if ((block.timestamp - start) <= delay()) revert ProposedOwnableFacet__delayElapsed_delayNotElapsed();
+    _;
+  }
```

Comment:
If the custom error type is not important in case of a not elasped delay, these `2` modifiers can be combined into one (we can just use `delayElapsed(uint256)`).